src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <CalculaorUI />
+  );
+}
+
+const CalculaorUI = () => {
+  const numbers = ['AC', '+/-', '%', 9, 8, 7, 6, 5, 4, 3, 2, 1, 0, '.'];
+  const operators = ['=', '+', '-', '*', '/'];
+
+  const numbersBtn = numbers.map((number) => (
+    <button className="btn" key={number} type="button">{number}</button>
+  ));
+
+  const operatorBtn = operators.map((operator) => (
+    <button className="btn" key={operator} type="button">{operator}</button>
+  ));
+
+  return (
+    <div className="app-container">
+      <div className="calculator">
+        <div className="display">0</div>
+        <div className="operator-numbers">
+          <div className="digits">{numbersBtn}</div>
+          <div className="operators">{operatorBtn}</div>
+        </div>
+      </div>
+    </div>
+  );
+};
```

Comment:
- [ ] @MMhaidari great work on your calculator component loved the simplicity, however, you forgot to create a child component to your Calculator component, you are supposed to create a child component before adding the component to top-level component `App.js`
![image](https://user-images.githubusercontent.com/61408860/228289545-0f6c77bc-a9f2-4974-b5a6-120428908afe.png)

Reply:
@danmainah dear reviewer the CalculatorUI itself is a child component which is the upper-level component which is Calculator function